### DISCUSSION TITLE
OSX的编辑模式字号太小，如何调节？

### DISCUSSION THREAD

**Discussion by dlguoming** [tamlok/vnote#1926] [General]
OSX 10.13，软件版本3.9。
外接2k显示器，在实际使用的时候编辑模式字体实在太小了，比阅读模式还小。
如何调节字号，以适应高分辨率显示器？

**Comment by tamlok** [comment]
按住command，然后鼠标滚轮。或者设置里面设置一下缩放。或者修改主题里面的`text-editor.theme`文件。

**Reply by dlguoming** [reply]
> 按住command，然后鼠标滚轮。或者设置里面设置一下缩放。或者修改主题里面的`text-editor.theme`文件。

感谢，我稍后试一下。

发现在OSX上遇到的问题挺多的......
比如tag的对话框我没办法输入有效字符（不知道是不是第三方输入法的锅），字都写进正文里了。

**Reply by tamlok** [reply]
> > 按住command，然后鼠标滚轮。或者设置里面设置一下缩放。或者修改主题里面的`text-editor.theme`文件。
> 
> 感谢，我稍后试一下。
> 
> 发现在OSX上遇到的问题挺多的...... 比如tag的对话框我没办法输入有效字符（不知道是不是第三方输入法的锅），字都写进正文里了。

我这两天上去看看。我只有虚拟机，所以基本只能靠你们反馈体验来修复了。

**Reply by dlguoming** [reply]
> > > 按住command，然后鼠标滚轮。或者设置里面设置一下缩放。或者修改主题里面的`text-editor.theme`文件。
> > 
> > 
> > 感谢，我稍后试一下。
> > 发现在OSX上遇到的问题挺多的...... 比如tag的对话框我没办法输入有效字符（不知道是不是第三方输入法的锅），字都写进正文里了。
> 
> 我这两天上去看看。我只有虚拟机，所以基本只能靠你们反馈体验来修复了。

我也是昨天才发现的，最近开始把joplin上的MD文件转存到本地，结果就发现没办法在tag对话框里键入文字。就赶紧反馈了，目前还不知道是不是搜狗输入法的问题。

**Reply by tamlok** [reply]
帮忙试一下原生输入法？

**Reply by dlguoming** [reply]
> 帮忙试一下原生输入法？

刚才开机试了试，目前发现的问题是这样婶儿的：

阅读模式——原生英文、拼音，搜狗输入法在标签对话框内可用，但只能键入英文，无法输入中文；
编辑模式——原生英文在标签对话框内可用。原生拼音、搜狗输入法输入内容会出现在正文中，标签对话框输入无效

设备：mbp pro 2011
OS：OSX 10.13.6
软件版本：3.9

**Reply by tamlok** [reply]
好的，谢谢。我这周看怎么修复。字号问题解决了吗？

**Reply by dlguoming** [reply]
> 好的，谢谢。我这周看怎么修复。字号问题解决了吗？

哈哈哈，感谢大大惦记。
我用您说的command键缩放解决的，这样还方便随时调节字体。

**Reply by tamlok** [reply]
已经修复输入法问题, 3.10.0

**Reply by dlguoming** [reply]
> 已经修复输入法问题, 3.10.0

收到，我这就去更新看看。
感谢。
用1.x/2.x好久，才更3.x还得慢慢摸索。
